### DISCUSSION TITLE
Copy OG Tags

### DISCUSSION THREAD

**Discussion by Ruriko** [YOURLS/YOURLS#3076] [Q&A] [answered]
I'm using the sample toolbar plugin which makes an iframe with a toolbar but I deleted the toolbar but kept the iframe. Basically the iframe hides the real url but I also want to make it auto generate og tags by copying the tags from the source site. How can I do this?

**Comment by ozh** [comment, marked as answer]
This is a PHP question more than a YOURLS question. Scrape remote page and pull some tags.

See https://github.com/YOURLS/YOURLS/wiki/Possible-With-A-Plugin
